themes/default/data/registry/packages/scaleway.yaml

```diff
@@ -1,14 +1,14 @@
-category: Infrastructure
+category: Cloud
 component: false
 description: A Pulumi package for creating and managing scaleway cloud resources.
 featured: false
-logo_url: "https://raw.githubusercontent.com/jaxxstorm/pulumi-scaleway/master/assets/scaleway-svgrepo-com.svg"
-name: scaleway
+logo_url: ""
+name: Scaleway
```

Comment:
This should remain as the package name so please revert this. The `title` property is the display name as called out by Christian in his [comment](https://github.com/pulumi/registry/pull/556/files#r785054269).